lib/vdsm/v2v.py

```diff
@@ -148,7 +148,7 @@ class JobNotDone(ClientError):
 
 
 class NoSuchOvf(V2VError):
-    ''' Ovf path is not exists in /run/vdsm/v2v/ '''
+    ''' Ovf path is not exists in /var/run/vdsm/v2v/ '''
```

Comment:
This change is incorrect. /var/run is deprecated for long time, and /run is correct.